en/developer_guide/elasticsearch_add_new_indexed_document.rst

```diff
@@ -0,0 +1,85 @@
+Elasticsearch add new indexed document
+======================================
+
+General note
+------------
+
+To use this documentation, you should already have installed `Elasticsearch`_ on your
+environment.
+
+This documentation will describe how to create the schema, and how to populate the index of
+the new document.
+
+Schema creation
+---------------
+
+To generate the schema of your document, you need to create an initializer that will implement
+``ElasticaSchemaInitializer``.
+
+This initializer should be tagged with :
+
+.. code-block:: yaml
+
+    tags:
+        - { name: open_orchestra_elastica.schema_initializer.strategy }
+
+This initializer will be called by the console command ``orchestra:elastica:schema:create``.
+
+This initializer can call a schema generator class that should implement
+``DocumentToElasticaSchemaGeneratorInterface``.
+
+Index population
+----------------
+
+To populate the index with your document data, there are two possible actions :
+
+* Populate the index from the data of you database
+* Modify the index each time one of your document is modified
+
+Populate from the database
+``````````````````````````
+
+To populate the index from the database, you need to create a populator that will implement
+``ElasticaPopulatorInterface``.
+
+This populator should be tagged with :
+
+.. code-block:: yaml
+
+    tags:
+        - { name: open_orchestra_elastica.populator.strategy}
+
+This populator will query all the documents that you want to index from the database. This
```

Comment:
Same remark here, what is the criteria for the manager to choose one or another strategy ?